src/tools/web/client/css/react-treeview.css

```diff
@@ -0,0 +1,42 @@
+/* the tree node's style */
+.tree-view {
+  overflow-y: hidden;
+}
+
+.tree-view_item {
```

Comment:
I copied this from [`react-treeview`](https://github.com/chenglou/react-treeview) (which I based my treeview off), but I'll change it to use `-` only.